crates/wasmparser/src/readers/core/types.rs

```diff
@@ -530,8 +548,10 @@ impl fmt::Display for RefType {
 /// is an invalid type.
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 pub enum HeapType {
-    /// Function of the type at the given index.
+    /// User defined type at the given index.
     TypedFunc(u32),
```

Comment:
Absolutely. I was thinking of doing a follow-up PR focusing specifically on that. Why not now? It would touch so much code, so it'd be hard to tell that the purpose of the PR is actually what it says in the title 😅

Reply:
Yeah, this is fine to do in the next PR, since so much stuff gets pulled into the renaming and needs to be adjusted.

Reply:
Added a TODO item for the follow-up.

Reply:
FWIW in the future I would personally request that the scattered `// TODO` comments not get committed with PRs. They have a way of not aging well over time, so I'd prefer if things like that are tracked in issues/notes/etc. (this is also one reason I'm more of a fan of big PRs that are complete rather than piecemeal PRs, but that's ok). It's ok to leave these in here for now though.